docs/learn/multiversx-ecosystem.md

```diff
@@ -0,0 +1,59 @@
+---
+id: multiversx-ecosystem
+title: What is MultiversX?
+---
+
+[comment]: # (mx-abstract)
+
+## **Summary**
+
+:::important
+
+MultiversX is a global network of interconnected computers that adhere to the [**MultiversX Protocol**](https://github.com/multiversx/mx-chain-go), providing a robust foundation for communities, applications, organizations, and digital assets. This decentralized platform enables users to create and manage their MultiversX accounts from anywhere in the world, at any time. 
+
+With **MultiversX**, you can explore a vast ecosystem of applications or develop your own, all without relying on a central authority that could alter the rules or limit your access. The core innovation of MultiversX lies in its trustless environment, ensuring that the network remains open, transparent, and secure for all participants.
+
+:::
+
+[comment]: # (mx-context-auto)
+
+## **What is MultiversX?**
+
+MultiversX is a distributed transactional computation protocol that relies on a sharded state architecture and a [Secure Proof of Stake](/docs/learn/consensus.md) consensus mechanism. While most other blockchain networks require custom hardware and high energy consumption, MultiversX runs on [average computers](/docs/validators/system-requirements.md).
+
+By employing [_sharding_](/docs/lea), a method of parallelizing data & transactions processing, MultiversX’s performance will scale up with the number of computers joining the network, reaching more than 100.000 transactions per second while growing increasingly decentralized.
+
+:::important
+**What is a blockchain?**
+
+A blockchain is a database of transactions that is continually updated and shared across numerous computers in a network. Each new set of transactions is grouped into a "block," which is added to the chain of previous transactions, hence the name blockchain. Public blockchains like MultiversX enable anyone to add data, but not remove it (**immutable**). To alter any information or manipulate the system, one would need to compromise the majority of computers on the network, which is an incredibly challenging task. This inherent difficulty makes decentralized blockchains like MultiversX highly secure and trustworthy.
```

Comment:
"which is added to the chain of previous transactions" -> maybe "previous blocks"?